applications/MeshMovingApplication/CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ endif(${KRATOS_BUILD_TESTING} MATCHES ON)
 ## define library Kratos which defines the basic python interface
 pybind11_add_module(KratosMeshMovingApplication MODULE THIN_LTO ${KRATOS_MESH_MOVING_APPLICATION_SOURCES} ${KRATOS_MESH_MOVING_TESTING_SOURCES})
 target_link_libraries(KratosMeshMovingApplication PRIVATE KratosCore )
+set_target_properties(KratosMeshMovingApplication PROPERTIES COMPILE_DEFINITIONS "MESH_MOVING_APPLICATION=EXPORT,API")
```

Comment:
is this needed?

Reply:
Not sure, but its ok to have it just in case

Reply:
it seems like this is the only file left in this PR (apart from the license which probably anyway should not be there...)
Why not copy those changes manually?

Reply:
:+1: for @philbucher's suggestion

Reply:
@philbucher Ideally the changes of this PR should be none. The problem is that with the current status of the master there are commits (not changes) that exists only in the release branch, making them incompatible. 
Also, one basing its development in Release 6.0 cannot merge master into or push fixes into both branches.
I did this to fix this situation. 

I would say that 90% of the merge conflicts end up solving themselves accepting the current because where basically modifying the same and probably the PR title is misleading as this is being done mostly to maintain the coherence rather than porting the fixes themselves.

I would like @pooyan-dadvand to confirm that this is working for him as well before merging.

Reply:
Also, in order this not to happen again I will change the way hotfixes in the next release are integrated into master as it seems this approach is problematic.

Reply:
Wouldn't a rebase of the problem branches be a cleaner solution?

Reply:
@roigcarlo I am not opposed to the changes, just wary of getting the file history corrupted by weird merges (say from merging a branch created from your H2 directly to HEAD), as has happened in the past. I believe this can be more or less avoided if you merge master to your branch before the final merge, though, so I would just ask you to be cautious with those.

Reply:
Only my commits should appear in the HEAD, rest will be merged in place at the time they were created, like if it were a normal PR integrating a branch, but with months of delay.

I can even squash them so my changes don't appear ans you only see a mostly empty merge ( with the exception of the licence file and the KRATOS_API in the meshmoving)

Reply:
It is not so much this PR I am worried about, but the branches based on 6.0 you say require this. I just think we should keep a close eye on those, since weird things tend to happen when bringing in old branches of branches.

Reply:
Agree on that, but @pooyan-dadvand will be the one using it (at least now), that's why I asked him